ksql-functional-tests/src/test/resources/query-validation-tests/sum.json

```diff
@@ -89,6 +89,25 @@
         {"topic": "S2", "key": 100, "value": "100,801.7"}
       ]
     },
+    {
+      "name": "sum decimal",
+      "statements": [
+        "CREATE STREAM TEST (ID bigint, NAME varchar, VALUE decimal(4,1)) WITH (kafka_topic='test_topic', value_format='DELIMITED', key='ID');",
```

Comment:
nit: remove NAME column as its just noise in the test.

Reply:
I will do this in the same follow-up as above since and just change the whole file